core/src/navigation/services/header.js

```diff
@@ -23,7 +23,9 @@ export const processHeaderSettings = component => {
 
     // Set Favicon
     if (header.favicon) {
-      const isInvalidIcoFile = !header.favicon.split('?')[0].endsWith('.ico');
+      const isInvalidIcoFile =
```

Comment:
Please rename the variable to e.g. isInvalidFormat or isInvalidFaviconFormat

Reply:
done